services/controllerhost/replicaStatsMRUCache.go

```diff
@@ -127,82 +132,134 @@ func (c *replicaStatsMRUCache) get(extent extentID, store storeID) (val *shared.
 		if len(foundStore) != 0 {
 			lclLg.WithField(`substituteStore`, common.FmtStor(string(foundStore))).
 				Error(`Could not get stats for specified store; substituting`)
-			return c.get(extent, foundStore)
+			return psmap.m[foundStore] // Might be nil
 		}
 
 		lclLg.Error(`Could not get stats for specified store`)
 		return nil
-	} else if val != nil {
-		c.put(extent, store, val)
+	} else if extStats != nil {
+		val = makeCompactExtentStats(extStats)
+		// In the typical case, all store extents had planned sealing, meaning that their available sequences are the same
+		// Map the value pointer to the same compactExtentStats in this typical case, to reduce memory usage by nearly 66%
+	dedup:
+		for dedupStore, val2 = range psmap.m {
+			if store != dedupStore && val2 != nil && val2.seqAvailable == val.seqAvailable {
```

Comment:
It's the same extent, but we have multiple copies of the store extent because the particular store that we read from is important in the unplanned seal case. We would otherwise get 'off by a few' phantom backlog.

This optimization reduces memory use for the normal case that all replicas for a sealed extent agree on the available sequence.